controllers/object/SplatTask.js

```diff
@@ -62,13 +65,23 @@ class SplatTask {
                 method: 'POST',
                 headers: {
                     ...form.getHeaders(),
+                    Authorization: `Bearer ${this.userJWT}`,
                 },
                 body: form,
             });
 
-            const gaussianSplatRequestId = await res.text();
-            this.object.gaussianSplatRequestId = gaussianSplatRequestId;
-            this.gaussianSplatRequestId = gaussianSplatRequestId;
+            let responseText = await res.text();
```

Comment:
This is another place where an error could happen and should probably be wrapped in a try/catch of its own